### ISSUE TITLE
Codepen- Mediapipe Pose code not working

### ISSUE THREAD

**Issue by Shubhodeepdas83** [google/mediapipe#2432]
<em>Please make sure that this is a [solution](https://google.github.io/mediapipe/solutions/solutions.html) issue.<em>

**System information** (Please provide as much relevant information as possible)
- Have I written custom code (as opposed to using a stock example script provided in Mediapipe): **NO**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04, Android 11, iOS 14.4): **Windows , Android
- [MediaPipe version](https://github.com/google/mediapipe/releases):**
- Bazel version:
- Solution (e.g. FaceMesh, Pose, Holistic): **Pose**
- Programming Language and version ( e.g. C++, Python, Java):

I am trying to checkout the Mediapipe Pose estimation JavaScript code uploaded on Codepen (https://codepen.io/mediapipe/pen/jOMbvxw)
But am consistently getting few errors. I tried running on Firefox , Edge and Chrome on Windows and Android.

Errors are :
1 ) 
Uncaught (in promise) RuntimeError: abort(Module.arguments has been replaced with plain arguments_ (the initial value can be provided on Module, but after startup the value is only looked for on a local variable of that name)) at Error
at jsStackTrace (pose_solution_simd_wasm_bin.js:9)
    at stackTrace (pose_solution_simd_wasm_bin.js:9)
    at abort (pose_solution_simd_wasm_bin.js:9)
    at Object.get (pose_solution_simd_wasm_bin.js:9)
    at pose_solution_simd_wasm_bin.js:9
    at ka.h (pose.js:35)
    at V (pose.js:13)
    at ma.next (pose.js:14)
    at pose.js:25
    at new Promise (<anonymous>)
    at abort (pose_solution_simd_wasm_bin.js:9)
    at Object.get (pose_solution_simd_wasm_bin.js:9)
    at pose_solution_simd_wasm_bin.js:9
    at ka.h (pose.js:35)
    at V (pose.js:13)
    at ma.next (pose.js:14)
    at pose.js:25
    at new Promise (<anonymous>)
    at Y (pose.js:25)
    at pose.js:35
2) 
pose_solution_simd_wasm_bin.js:9 Uncaught (in promise) TypeError: Cannot read property 'buffer' of undefined
    at Object.write (VM21 pose_solution_simd_wasm_bin.js:9)
    at Object.write (VM21 pose_solution_simd_wasm_bin.js:9)
    at Object.createDataFile (VM21 pose_solution_simd_wasm_bin.js:9)
    at finish (VM21 pose_solution_simd_wasm_bin.js:9)
    at processData (VM21 pose_solution_simd_wasm_bin.js:9)
    at Object.createPreloadedFile [as FS_createPreloadedFile] (VM21 pose_solution_simd_wasm_bin.js:9)
    at DataRequest.finish (VM20 pose_solution_packed_assets_loader.js:126)
    at DataRequest.onload (VM20 pose_solution_packed_assets_loader.js:121)
    at processPackageData (VM20 pose_solution_packed_assets_loader.js:157)
    at runWithFS (VM20 pose_solution_packed_assets_loader.js:168)
3)
Uncaught (in promise) RuntimeError: abort(undefined) at Error
    at jsStackTrace (VM26 pose_solution_simd_wasm_bin.js:9)
    at stackTrace (VM26 pose_solution_simd_wasm_bin.js:9)
    at abort (VM26 pose_solution_simd_wasm_bin.js:9)
    at _abort (VM26 pose_solution_simd_wasm_bin.js:9)
    at pose_solution_simd_wasm_bin.wasm:0x4d5c46
    at pose_solution_simd_wasm_bin.wasm:0x4d5afe
    at pose_solution_simd_wasm_bin.wasm:0x4d5cbb
    at pose_solution_simd_wasm_bin.wasm:0x1adda
    at pose_solution_simd_wasm_bin.wasm:0x14c31
    at SolutionWasm$send [as send] (eval at new_ (VM26 pose_solution_simd_wasm_bin.js:9), <anonymous>:9:1)
    at abort (VM26 pose_solution_simd_wasm_bin.js:9)
    at _abort (VM26 pose_solution_simd_wasm_bin.js:9)
    at pose_solution_simd_wasm_bin.wasm:0x4d5c46
    at pose_solution_simd_wasm_bin.wasm:0x4d5afe
    at pose_solution_simd_wasm_bin.wasm:0x4d5cbb
    at pose_solution_simd_wasm_bin.wasm:0x1adda
    at pose_solution_simd_wasm_bin.wasm:0x14c31
    at SolutionWasm$send [as send] (eval at new_ (VM26 pose_solution_simd_wasm_bin.js:9), <anonymous>:9:1)
    at ka.h (pose.js:47)
    at V (pose.js:13)

Is there any work around to make this work? 
Pointing to previous versions at Mediapipe/pose CDN does seem to work for laptop browser mode, but those models dont work for android browser.

Thanks

**Comment by mhays-google** [comment]
tl;dr; -- Refresh your cache.

I got the same error and realized that the Pose CodePen got a code change earlier today (today for my time zone) that relies on a non-breaking change in the latest version of the pose library.  I probably should have just made it a breaking change, but this problem should clear up in the next 24 hours.

If you refresh your cache and it still happens, write to me here and I'll check tomorrow morning (California time).

**Comment by sgowroji** [comment]
Hi @Shubhodeepdas83, Could you please try clearing the cache and verify the above. That works.

**Comment by Shubhodeepdas83** [comment]
Hi Michael, Appreciate your quick response.
I tried the refreshing the cache as you suggested. The problem still persists intermittently. It throws error around 40% of time you access it. And once it goes in error mode, even multiple refreshing does not help.

P.S : I have been working on Mediapipe/Pose for last 2-3 months. This problem appeared around 1-2 months back, i guess around 0.4 release. We simply pointed to 0.3 version on the CDN to resolve. But new version seems to be more compatible on mobile browser, so wanted to check if we can make it work.

**Comment by mhays-google** [comment]
Sorry you are having this issue...

The reason you aren't seeing this issue in 0.3 is almost certainly because
I never update 0.3.  :-)

I'm having discussions with the group on how to better use semantic
versioning to prevent these breaks from interfering.  If I can convince the
team to let me increment the middle number with every push, this will go
away.  So, to that point, I believe that the issue is almost certainly
caching.  I use jsdelivr to get files from NPM, and it has a long cache
timeout.  I've started using a script to send purge instructions
automatically to jsdelivr when I push a version, and in these cases, it
then still leaves it up to your local browser's caching mechanism to
resolve.

However, my recommendation is to explicitly lock the version to exactly
what you want. e.g, ***@***.***

If you are using a service like jsnotify, you will need to change you
`locateFile` method as well.  To make this easier to deal with, I have
started exposing a VERSION variable from the module that will give you the
exact version imported by the script.  In the demo for pose, you'll see
this code:

const options = {
  locateFile: (file) => {
  return `
***@***.***/pose@${mpPose.VERSION}/${file}`;
}};

**Comment by Shubhodeepdas83** [comment]
Hi Michael,
We tried lots of different things. Different browsers on HP, Lenovo, Ipad, Andorid , CPU only machines, GPU machines.
Just to take the Cache out of equation , we even tried downloading the NPM files to Local machine, installing a new Chrome, and then using the Codepen code , it stills gives the same errors.
Version 0.3.xx works, so the device specs should be good enough.
Are we missing something? 
Apologies for bugging you so much for this.

**Comment by sureshdagooglecom** [comment]
Hi @Shubhodeepdas83 ,
This is issue can be resolved in the latest version of MediaPipe , could you please check and close this issue.

**Comment by id1945** [comment]
It seems that google has not been able to fix this error. This is a dangerous error and limits developers to difficult problem logic situations 
![images](https://user-images.githubusercontent.com/40824445/198860994-5397a6b4-d5db-4d16-a44f-2fbc4e969c06.png)
